tests/integration/IntegrationSpec.hs

```diff
@@ -60,12 +62,14 @@ main = runSimpleApp $ do
                      <> fromString (takeFileName next)
               res <- test next
               loop (idx + 1) rest' (res <> accum)
-
+    
     loop (1 :: Int) (Set.toList tests) mempty
-
+  finalTime <- getCurrentTime
   let (successes, failures) = partition ((== ExitSuccess) . snd)
                             $ Map.toList results
-
+  let timeDiff = diffUTCTime finalTime startTime
+  let timeDiffStr = formatTime defaultTimeLocale "%h:%m:%s" timeDiff
```

Comment:
Did you mean `"%h:%m:%s"`, or was your intention `"%H:%M:%S"`. The former generates output like:
~~~text
2024-06-22T16:28:37.8924296Z Integration tests ran in : 0:19:1191
~~~ 

because 1,191 seconds are equivalent to about 19 whole minutes and about 0 whole hours.

Reply:
I switched to "%H:%M:%S - total %s seconds"

Reply:
(I misunderstood the docs)